src/Compilers/Core/Portable/Operations/IIsNullOperation.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+namespace Microsoft.CodeAnalysis.Operations
+{
+    /// <summary>
+    /// Represents result of checking whether the <see cref="Operand"/> is null. 
```

Comment:
:bulb: `<see langword="null"/>` on this one. You can press <kbd>Ctrl</kbd>+<kbd>Space</kbd> on the existing word `null` to expand it to this form.